src/services/datadog.ts

```
@@ -31,7 +31,7 @@ async function log(
   }
 
   const body = ctx.req.header("content-type")?.startsWith("application/json")
-    ? await ctx.req.raw.clone().json()
+    ? await ctx.req.json()
```

are you sure you don't want to wrap all this in a `try-catch`? :smile: 

We keep breaking production with something that meant is to help us not break production :wink:

---

I think it's in a waitUntil, so should be fine. It was silently failing in the background before